app/models/manageiq/providers/azure_stack/inventory/persister/definitions/cloud_collections.rb

```diff
@@ -21,4 +22,10 @@ def add_resource_groups
       builder.add_default_values(:ems_id => manager.id)
     end
   end
+
+  def add_orchestration_stacks
+    add_collection(cloud, :orchestration_stacks) do |builder|
+      builder.add_properties(:model_class => ::ManageIQ::Providers::AzureStack::CloudManager::OrchestrationStack)
```

Comment:
@miha-plesko was this not being picked up correctly?  It should be able to determine this automatically